app/helpers/form_helper.rb

```diff
@@ -232,12 +232,7 @@ def autocomplete_f(f, attr, options = {})
   end
 
   def byte_size_f(f, attr, options = {})
-    options[:class] = options[:class].to_s + ' byte_spinner' unless options[:disabled]
-    options[:label_help] = _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified.")
-    options[:help_block] ||= soft_limit_warning_block
-    options[:help_block] += f.hidden_field(attr, :class => "real-hidden-value", :id => nil)
-
-    text_f(f, attr, options)
+    react_form_input('memory', f, attr, options)
```

Comment:
looks good to me, but maybe plugin owners that use it can help test if it also works for them?
@tristanrobert mind take a look for [foreman_fog_proxmox](https://github.com/theforeman/foreman_fog_proxmox/blob/ca9475c15731e866cc3d8b1998320b30838f16d1/app/views/compute_resources_vms/form/proxmox/container/_config.html.erb#L31), there are several usages there
@shiramax mind take a look for [foreman_kubevirt](https://github.com/theforeman/foreman_kubevirt/blob/ab687633922292a13e2f97720d2df86fc153723c/app/views/compute_resources_vms/form/kubevirt/_base.html.erb#L7)?

If those are the only use cases, @ezr-ondrej how about using only `react_form_input` so we won't need to maintain another wrapper?

Reply:
I think the wrapper is very simple and beneficial - now we are free to change the implementation and the users are not required to do any changes in their code, if we would decided we want to implement this component in angular in the future, we can do so.